Explorer/Assets/DCL/Notifications/NotificationEntry/NotificationView.cs

```diff
@@ -0,0 +1,41 @@
+using DCL.UI;
+using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+namespace DCL.Notification.NotificationEntry
+{
+    public class NotificationView : MonoBehaviour, IPointerClickHandler
+    {
+        public event Action<NotificationType> OnNotificationClicked;
```

Comment:
Following our code-style, we don't use `On` prefix for events, since events named as finished/eccured event - just verb in past. 
`public event Action<NotificationType> NotificationClicked`
Then subscription looks nicer (because we can use `On-` prefix there)
`myNotificationView.NotificationClicked += OnNotificationClicked`